File: Learnify.Domain/Entities/Certificate.cs
namespace Learnify.Domain.Entities
{
    public class Certificate : BaseEntity
    {
        public Guid StudentId { get; set; }
        public User? Student { get; set; }
        public Guid CourseId { get; set; }
        public Course? Course { get; set; }
        public DateTime IssuedAt { get; set; } = DateTime.UtcNow;
    }
}

File: Learnify.UI/src/models/entities/Certificate.js
import { BaseEntity } from "./BaseEntity";

export class Certificate extends BaseEntity {
    constructor(id, studentId, student = null, courseId, course = null, issuedAt) {
        super(id);
        this.studentId = studentId;
        this.student = student;
        this.courseId = courseId;
        this.course = course;
        this.issuedAt = issuedAt;
    }
}